betterdgg/modules/emoticons.js

```diff
@@ -179,6 +188,40 @@
                 if (override) {
                     wrapped.find('.chat-emote').addClass('bdgg-chat-emote-override');
                 }
+            },
+            organizeEmotes: function() {
+                // Show the emote menu
+                $('#emoticon-btn').click();
+                
+                var $emotesBox = $('#destiny-emotes');
+                
+                // Delete the spacing because I personally think it's ugly
+                $emotesBox.parent().find('hr').remove();
+                
+                $emotesBox.prepend('<div id="TwitchEmoteContainer"><h6>Twitch Emotes (Non Sub)</h6></div>');
+                $emotesBox.prepend('<div id="BTTVEmoteContainer"><h6>BTTV Emotes</h6></div>');
+                $emotesBox.prepend('<div id="BBDGGEmoteContainer"><h6>BBDGG Emotes</h6></div>');
+                $emotesBox.prepend('<div id="DGGEmoteContainer"><h6>DGG Emotes</h6></div>');
+                
+                $emotesBox.find('.chat-emote').each(function(id, elem) {
+                    var $emoteName = $(elem).attr('title');
+                    var $emoteClone = $(elem).parent().clone();
+                    
+                    // Remove old emote
+                    $(elem).parent().remove();
+                    
+                    // Filter emotes into their respective containers
+                    if ($.inArray($emoteName, EMOTICONS) > -1) { $('#DGGEmoteContainer').append($emoteClone); }
+                    else if ($.inArray($emoteName, TWITCH_EMOTICONS) > -1) { $('#TwitchEmoteContainer').append($emoteClone); }
+                    else if ($.inArray($emoteName, BTTV_EMOTICONS) > -1) { $('#BTTVEmoteContainer').append($emoteClone); }
+                    else if ($.inArray($emoteName, BBDGG_EMOTICONS) > -1) { $('#BBDGGEmoteContainer').append($emoteClone); }
+                    else if ($.inArray($emoteName, ANIMATED) > -1) { $('#BBDGGEmoteContainer').append($emoteClone); }
+                    else if ($.inArray($emoteName, SUBONLY) > -1) { null; }
```

Comment:
I would have preferred leaving the block empty but JS Lint throws me this  
  
`[16:51:41] Starting 'lint'...
[16:51:42] 'lint' errored after 820 ms
[16:51:42] ESLintError in plugin 'gulp-eslint'
Message:
    Empty block statement.
Details:
    fileName: G:\Projects\Better Better Destiny GG\Fork\betterdgg\betterdgg\modules\emoticons.js
    lineNumber: 219
`  
So I just put null in just to make JS Lint happy. The sub emotes are already in their own container in the bottom so I just decided to leave them there and delete the duplicate ones.